src/wase/core/terminate_exception.h

```diff
@@ -11,7 +11,7 @@ class TerminateException : public std::exception
 public:
 	TerminateException(const std::string& message) : message(message) {}
 
-	const char* what() const
+	const char* what() const noexcept override
```

Comment:
Could you explain this line? std::exception has no what method so what are we trying to override?

Reply:
This is about MinGW build too. I guess we don't need override. But we need noexcept for build without error.

Reply:
Yeah just remove override then its fine, thanks for the clarification.

Reply:
We need override keyword. Because std::exception have a virtual what function.

https://en.cppreference.com/w/cpp/error/exception
![image_2021-07-22_005101](https://user-images.githubusercontent.com/12533045/126564882-1cd1d7aa-6e4a-48d5-8d97-8d7a4bbacb65.png)